### PR TITLE
update pi-apps description and location

### PR DESCRIPTION
**Pull request by theofficialgman**
in line with other debian/ubuntu (x86_64) packages

Signed-off-by: theofficialgman <28281419+theofficialgman@users.noreply.github.com>

### CODE REVIEW

**Comment by getchoo** [comment]
i'm not sure why the icon was changed, as this is specific to the raspberry pi and looks a little odd next to another debian icon. i think just having "raspberry pi" as the title before looked a little better as well, but i can see this how this would add clarification

**Comment by Scrumplex** [comment]
Needs rebase/merge

**Comment by theofficialgman** [comment]
> i'm not sure why the icon was changed, as this is specific to the raspberry pi and looks a little odd next to another debian icon. i think just having "raspberry pi" as the title before looked a little better as well, but i can see this how this would add clarification

I am a maintainer and developer at pi-apps. No it isn't specific to the pi one bit. It wasn't even built and packaged using one (used my Nintendo switch).
Pi-apps is simply the project name. It supports any arm system (eg: Nvidia Jetson, raspberry pi, odroid, banana pi, etc).

**Comment by theofficialgman** [comment]
@Scrumplex rebased https://theofficialgman.github.io/prismlauncher.github.io/download/linux/

File: src/download/linux.md [hunk 1]
```diff
@@ -129,6 +129,18 @@ una install prismlauncher-git
...
+</div>
+  
+<div class="infobox top">
+
+# <img src="https://www.vectorlogo.zone/logos/debian/debian-icon.svg" height="20" /> Debian / <img src="https://www.vectorlogo.zone/logos/raspberrypi/raspberrypi-icon.svg" height="20" /> Pi OS / <img src="https://www.vectorlogo.zone/logos/ubuntu/ubuntu-icon.svg" height="20" /> Ubuntu (ARM32/64)
+
+Prism Launcher is avalible in the [pi-apps](https://github.com/Botspot/pi-apps) store as a deb install.
+  
+NOTE: Only supports Debian/Raspbian/Pi OS Buster and newer and Ubuntu 18.04 and newer.
+*Debian based ARM packages are community-maintained, Prism Launcher is not responsible for outdated versions.*
```
Comment (DioEgizio) [inline src/download/linux.md:142]:
```suggestion
*Debian-based ARM packages are community-maintained, Prism Launcher is not responsible for outdated versions.*
```

**Review by Scrumplex — APPROVED** [review]

**Comment by Scrumplex** [comment]
Thanks!

**Pull request merged**